bin/v-add-user

```diff
@@ -274,6 +274,9 @@ fi
 # Adding jailed sftp env
 $BIN/v-add-user-sftp-jail $user
 
+# Adding jailed ssh env
+$BIN/v-add-user-ssh-jail $user
```

Comment:
Jailed ssh user  should probably not enabled by default It should be at least an option to be enabled / disabled on demand

Reply:
On line: v-add-user-ssh-jail 41 only bash users allow to use it ... So it should be fine

Reply:
Note to self: Flag check still needs to be added here.